Src/PerformanceCollector/Shared/Implementation/IPerformanceCollector.cs

```diff
@@ -25,12 +25,12 @@ internal interface IPerformanceCollector
         /// <summary>
         /// Registers a counter using the counter name and reportAs value to the total list of counters.
         /// </summary>
-        /// <param name="perfCounterName">Name of the performance counter.</param>
+        /// <param name="perfCounter">Name of the performance counter.</param>
```

Comment:
its a name, but its not the counter name. Counter name is a small subset of the performance counter, which is why I renamed it to an abstract convention.